ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/caseActions/ntiUnderConsideration/editNtiUnderConsiderationPage.ts

```diff
@@ -1,9 +1,67 @@
 import { Logger } from "../../../common/logger";
 
-export class EditNtiUnderConsiderationPage
-{
-    public save(): this
+export class EditNtiUnderConsiderationPage {
+    public withReasons(value: string): this {
```

Comment:
Being picky here - should this be plural withReasons or singular withReason?

Reply:
I think you are right, it should be reason